_rules/SC1-2-1-audio-transcript.md

```diff
@@ -20,7 +20,7 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is:
+The rule applies to every [non-streaming](#non-streaming-media-element) `audio` element that is:
 - playing; or,
 - has a "play button" that is [visibile](#visible) and [included-in-the-accessibility-tree](#included-in-the-accessibility-tree).
 
```

Comment:
For the expectation below (sorry for not catching this earlier), I suggest rewording to:

> The auditory information of each test target is available through a text transcript. That text transcript is [visibile](#visible) and [included-in-the-accessibility-tree](#included-in-the-accessibility-tree), either on the page or through a link.

Main change is to remove the "needs to" because that does not match the wording for an expectation. Also minor editorial re-arrangement for readability.